scripts/build_server.sh

```diff
@@ -0,0 +1,10 @@
+echo "Build server files..."
+echo "Building CoffeeScript..."
+./node_modules/.bin/coffee -cb --output build/server server
```

Comment:
Tip: I recommend your script override the `$PATH` env variable (see https://github.com/cozy-labs/konnectors/pull/546/commits/c066b61363976a5b283f110e0a3136ffd30a3619#diff-d99b26f864526f6f8f79861d49deb922R3), so you command call can simply be `coffee -cb ...` (without the local path)